build fails on MacOS 14.3 arm64 The link interface of target "opencv_world" contains:      Eigen3::Eigen    but the target was not found.

### Is there an existing issue for this problem?

- [X] I have searched the existing issues

### OrcaSlicer Version

226450e

### Operating System (OS)

macOS

### OS Version

14.3.1

### Additional system information

arm64
latest XCode

### Printer

Bambu P1P

### How to reproduce

git clone https://github.com/SoftFever/OrcaSlicer.git
cd OrcaSlicer
./build_release_macos.sh

### Actual results

CMake Error at deps/build_arm64/OrcaSlicer_dep_arm64/usr/local/lib/cmake/opencv4/OpenCVModules.cmake:81 (set_target_properties):
  The link interface of target "opencv_world" contains:

    Eigen3::Eigen

  but the target was not found.  Possible reasons include:

    * There is a typo in the target name.
    * A find_package call is missing for an IMPORTED target.
    * An ALIAS target is missing.

Call Stack (most recent call first):
  deps/build_arm64/OrcaSlicer_dep_arm64/usr/local/lib/cmake/opencv4/OpenCVConfig.cmake:126 (include)
  src/libslic3r/CMakeLists.txt:479 (find_package)

### Expected results

build succeeds 

### Project file & Debug log uploads

not relevant

### Checklist of files to include

- [ ] Log file
- [ ] Project file

### Anything else?

_No response_

--- a/deps/OpenCV/OpenCV.cmake
+++ b/deps/OpenCV/OpenCV.cmake
@@ -26,7 +26,7 @@ orcaslicer_add_cmake_project(OpenCV
        -DBUILD_ZLIB=OFF
        -DWITH_1394=OFF
        -DWITH_CUDA=OFF
-       -DWITH_EIGEN=ON
+       -DWITH_EIGEN=OFF
        ${_use_IPP}
        -DWITH_ITT=OFF
        -DWITH_FFMPEG=OFF

